wrapping/src/openmeeg_path_setter.py

```diff
@@ -0,0 +1,26 @@
+#
+# find openmeeg lib
+#
+import sys
+import os
+
+# import locally built openmeeg
+topdir = os.getcwd()
+omdir = ""
+
+# heuristic
```

Comment:
why are we doing this adhoc path updating?

Reply:
Sorry, this is my mistake. This heuristic allows us to run the python script from anywhere inside the OpenMeeg topdir without any extra environment variable... (the scirpt basically find the openmeeg topdir, and then finds the data and the _openmeeg.so file). I didn't check it properly then running the test. I guess I just broke the python tests.

This script should be removed to get the old behavior back.